_run/common-kind.mk

```diff
@@ -75,10 +75,12 @@ kind-ingress-setup:
 	kubectl apply -f "$(INGRESS_CONFIG_PATH)"
 	"$(AKASH_ROOT)/script/setup-kind.sh"
 
+
 .PHONY: kind-cluster-delete
 kind-cluster-delete:
 	kind delete cluster --name "$(KIND_NAME)"
 
 .PHONY: kind-cluster-clean
 kind-cluster-clean:
-	kubectl delete ns -l akash.network
+	kubectl delete ns -f akash-services leases akash.network
+	# TODO: 'akash.network' used?
```

Comment:
`-l akash.network` means match all namespaces (here) that have a label named `akash.network`.

Reply:
Ahhh I see, that works